iaso/api/mobile/metadata/last_updates.py

```diff
@@ -0,0 +1,75 @@
+from django.db.models import Max
+from django.shortcuts import get_object_or_404
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status, permissions
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+
+from iaso.api.query_params import APP_ID
+from iaso.models import Form, FormVersion, OrgUnit, OrgUnitType, Project
+
+
+class LastUpdatesViewSet(ViewSet):
+    """Metadata Last Updates
+
+    This API is open to anonymous users:
+
+    /api/mobile/metadata/lastupdates/
+
+    `GET /api/mobile/metadata/lastupdates/?app_id=some.app.id`
+    """
+
+    permission_classes = [permissions.IsAuthenticatedOrReadOnly]
+    http_method_names = ["get", "head", "options"]
+    lookup_url_kwarg = [APP_ID]
+
+    app_id_param = openapi.Parameter(
+        name=APP_ID,
+        in_=openapi.IN_QUERY,
+        required=True,
+        description="Application id",
+        type=openapi.TYPE_STRING,
+    )
+
+    @swagger_auto_schema(
+        responses={
+            200: "provides the latest updated dates",
+            404: "project for given app id doesn't exist",
+        },
+        manual_parameters=[app_id_param],
+    )
+    def list(self, request: Request):
+        app_id = request.query_params.get(APP_ID)
+
+        forms = Form.objects
+        form_versions = FormVersion.objects
+        org_units = OrgUnit.objects
+        org_unit_types = OrgUnitType.objects
+        if app_id is not None:
```

Comment:
if we don't filter on app_id, we should still filter on user no?

Reply:
I thought it didn't make sense but I might totally be wrong